src/cli.zig

```diff
@@ -383,7 +384,14 @@ pub const Arguments = struct {
                     };
                 }
             }
-            ctx.test_options.update_snapshots = args.flag("--update-snapshots");
+            if (args.option("--bail")) |bail| {
+                if (bail.len > 0) {
+                    ctx.test_options.bail = std.fmt.parseInt(u32, bail, 10) catch |e| {
+                        Output.prettyErrorln("--bail expects a number: {s}", .{@errorName(e)});
```

Comment:
Should add a check that `bail > 0`, since `bail = 0` doesn't make sense

Reply:
Oh yeah was a little typo

Reply:
Wait, it checks for length of the parameter, so if it doesn't exist

Reply:
other numbers do it too

Reply:
oh, i just realized by what you mean. sure i'll do it